### ISSUE TITLE
Conversion to Dalvik format failed with error 1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [khj0704/android-query#38]
```
What steps will reproduce the problem?
1. import android-query.jar into projects libs folder
2. run project


What is the expected output? What do you see instead?
expected: application starts
what I see: Conversion to Dalvik format failed with error 1


What version of the product are you using? On what operating system?
android-query version: 21.5
operating system: windows 7 64 bit
eclipse indigo service release 2
Android Development Tools 18.0.0.v201203301601-306762


Please provide any additional information below.

Console output:

[2012-04-11 09:10:40 - BonBons] Dx 
EXCEPTION FROM SIMULATION:
[2012-04-11 09:10:40 - BonBons] Dx local variable type mismatch: attempt to set 
or access a value of type int using a local variable of type 
android.view.ViewGroup. This is symptomatic of .class transformation tools that 
ignore local variable information.

[2012-04-11 09:10:40 - BonBons] Dx ...at bytecode offset 00000062
locals[0000]: I
locals[0001]: Landroid/view/View;
locals[0002]: Landroid/view/ViewGroup;
locals[0003]: Ljava/lang/String;
locals[0004]: I
stack[top0]: I
...while working on block 0060
...while working on method 
shouldDelay:(ILandroid/view/View;Landroid/view/ViewGroup;Ljava/lang/String;)Z
...while processing shouldDelay 
(ILandroid/view/View;Landroid/view/ViewGroup;Ljava/lang/String;)Z
...while processing com/androidquery/util/Common.class

[2012-04-11 09:10:42 - BonBons] Dx 1 error; aborting
[2012-04-11 09:10:42 - BonBons] Conversion to Dalvik format failed with error 1
```

Original issue reported on code.google.com by `tome.a...@gmail.com` on 11 Apr 2012 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
I have exactly the same problem and cannot build my project with the latest 
aQuery release (0.21.5). Old version 0.20.6 works fine.
```

Original comment by `christo...@beyls.net` on 12 Apr 2012 at 4:08

**Comment by GoogleCodeExporter** [comment]
```
Same issue, with similar set up as original post.  I reverted back to 0.20.6.
```

Original comment by `jflo...@phunware.com` on 12 Apr 2012 at 10:41

**Comment by GoogleCodeExporter** [comment]
```
Hi. I have the same problem. No problem with android-query-full.0.21.5 by the 
way.
```

Original comment by `kaloskag...@gmail.com` on 15 Apr 2012 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 0.21.6. It's caused by some obscure bug with proguard. The 
core jar won't be pre-processed by proguard for now.

Thanks!
```

Original comment by `tinyeeliu@gmail.com` on 16 Apr 2012 at 2:07
- Changed state: **Fixed**
